### ISSUE TITLE
panicked at 'Failed to read directory: '"/Users/.../.cargo/git/checkouts/.DS_Store"'

### ISSUE THREAD

**Issue by upsuper** [matthiaskrgr/cargo-cache#65]
I'm on macOS. Probably the file is not expected to be there, but the program probably shouldn't panic anyway.

```
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
   1: std::panicking::default_hook::{{closure}}
   2: std::panicking::rust_panic_with_hook
   3: std::panicking::continue_panic_fmt
   4: std::panicking::begin_panic_fmt
   5: cargo_cache::cache::git_checkouts::GitCheckoutCache::checkout_folders
   6: rayon_core::join::join_context::{{closure}}::{{closure}}
   7: <rayon_core::job::StackJob<L,F,R> as rayon_core::job::Job>::execute
   8: rayon_core::registry::WorkerThread::wait_until_cold
```

**Comment by matthiaskrgr** [comment]
Thanks!
I already fixed this issue for registries some time ago but for some reason did not think it could appear in the git cache as well....

Can you verify that https://github.com/matthiaskrgr/cargo-cache/commit/e74094e2fee869efaf1bdea9d22869c411e02a72 fixes the issue for you?

**Comment by upsuper** [comment]
I tried installing from git master directly, and this issue seems to have been fixed, likely because of your fix. Thanks!

**Comment by matthiaskrgr** [comment]
Thanks, I released 0.3.1 which contains the fix!
